packages/console/src/containers/AppContent/index.tsx

```diff
@@ -1,38 +1,46 @@
 import { conditional, joinPath } from '@silverhand/essentials';
-import { useRef } from 'react';
+import { useContext, useRef } from 'react';
 import { Navigate, Outlet, useParams } from 'react-router-dom';
 
 import AppLoading from '@/components/AppLoading';
 import MauExceededModal from '@/components/MauExceededModal';
 import PaymentOverdueModal from '@/components/PaymentOverdueModal';
+import Topbar from '@/components/Topbar';
 import { isCloud } from '@/consts/env';
+import { TenantsContext } from '@/contexts/TenantsProvider';
 import useScroll from '@/hooks/use-scroll';
 import useUserPreferences from '@/hooks/use-user-preferences';
 
 import { getPath } from '../ConsoleContent/Sidebar';
 import { useSidebarMenuItems } from '../ConsoleContent/Sidebar/hook';
 
-import Topbar from './components/Topbar';
+import TenantSuspendedPage from './TenantSuspendedPage';
 import * as styles from './index.module.scss';
 import { type AppContentOutletContext } from './types';
 
 export default function AppContent() {
   const { isLoading } = useUserPreferences();
+  const { currentTenant } = useContext(TenantsContext);
+  const isTenantSuspended = isCloud && currentTenant?.isSuspended;
```

Comment:
nit: technically no need to check `isCloud` here

Reply:
This feature is cloud-only.
Although in the open-source version, the `isSuspended` field in the `currentTenant` data is set to `false`, but it is merely dummy data.
Therefore, we use `isCloud` here to ensure that this functionality does not accidentally activate due to changes in dummy data.